### DISCUSSION TITLE
Change the logo

### DISCUSSION THREAD

**Discussion by GaetanRdn** [stackblitz/tutorialkit#206] [Q&A]
I tried to change the logo of the application.

I used a .png whereas a .svg and it's not correctly rendered. It's rendered like an icon whatever the image size.

any idea?

**Comment by AriPerkkio** [comment]
This sounds like a bug. Feel free to open a [Bug report](https://github.com/stackblitz/tutorialkit/issues/new/choose) with reproducible setup.
